src/commands/environments/create.js

```diff
@@ -19,11 +20,8 @@ class CreateCommand extends AbstractAuthenticatedCommand {
         const errorData = JSON.parse(error.response.text);
         if (errorData && errorData.errors && errorData.errors.length
           && errorData.errors[0] && errorData.errors[0].detail) {
-          this.error(errorData.errors[0].detail, {
-            code: errorData.errors[0].status,
-            exit: 1,
-          });
-          return;
+          logger.error(errorData.errors[0].detail);
+          this.exit(1);
```

Comment:
[MISC] If you want to stay iso with what existed (and avoid the following `throw error`, you should return here.

Reply:
As discussed, after the `this.exit(1)`, all code added is flagged as unreachable by my IDE.
So let's keep it that way!